pkg/dispatchcli/cmd/manage.go

```diff
@@ -34,13 +34,13 @@ var (
 
 	manageExample = i18n.T(`
 # Enable Dispatch bootstrap mode while specifying bootstrap user
-dispatch manage --enable-bootstrap-mode --bootstrap-user admin@example.com -f ./config.yaml
```

Comment:
I think we should move the enable/disable bootstrap flags to their own subcommands in another PR.